### DISCUSSION TITLE
website is not getting open

### DISCUSSION THREAD

**Discussion by himanshubhatt1** [cssbattle/forum#115] [Bugs reports]
![error](https://user-images.githubusercontent.com/80446990/136020012-1e3d6900-fde3-4217-bb33-7f857108deb3.png)
In last few days i am not able to open CSS BATTLE , I have tried all my side, but no solution ,... please help...

**Comment by chinchang** [comment]
Can you try opening it on your mobile internet? might be your network problem.

**Reply by himanshubhatt1** [reply]
ok , thankn you
